scripts/cli.py

```diff
@@ -355,7 +355,14 @@ def deploy_to(controller, cloud, model, bundle, channel, public_address, build,
             ),
         )
 
-    juju('wait', '-wv', '-m', model, '-t', str(30 * 60))
+    for _ in range(60):
+        try:
+            juju('wait', '-wv', '-m', model, '-t', str(30 * 60), die=False)
+            break
+        except subprocess.CalledProcessError:
+            time.sleep(5)
+    else:
+        juju('wait', '-wv', '-m', model, '-t', str(30 * 60), die=False)
```

Comment:
Whats the context and reasoning behind this change?

Reply:
Good catch, updated with a comment with a link to the relevant bug report